SOPT-iOS/Projects/Features/HomeFeature/Sources/Cells/Announcements/AnnouncementCardCVC.swift

```diff
@@ -0,0 +1,177 @@
+    }
+    
+    private let contentLabel = UILabel().then {
+        $0.textColor = DSKitAsset.Colors.gray300.color
+        $0.font = DSKitFontFamily.Suit.medium.font(size: 14)
+        $0.lineBreakMode = .byTruncatingTail
+        $0.textAlignment = .left
+    }
+    
+    private let coverImageView = UIImageView().then {
+        $0.contentMode = .scaleToFill
+        $0.layer.cornerRadius = 8
+        $0.clipsToBounds = true
+    }
+    
+    private let contentStackView = UIStackView().then {
+        $0.axis = .vertical
+        $0.alignment = .leading
+        $0.spacing = 12
+    }
+    
+    // MARK: - Initialization
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setStackView()
+        setUI()
+        setLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+// MARK: - UI & Layout
+
+extension AnnouncementCardCVC {
+    private func setUI() {
+        self.backgroundColor = DSKitAsset.Colors.gray900.color
+        self.layer.cornerRadius = 12
+    }
+    
+    private func setLayout() {
+        self.addSubviews(contentStackView)
+        
+        writerProfileImageView.snp.makeConstraints { make in
+            make.size.equalTo(20)
+        }
+        
+        coverImageView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(160)
+        }
+        
+        contentLabel.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+        }
+        
+        contentStackView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(16)
+            make.top.equalToSuperview().inset(14)
+        }
+    }
+    
+    private func setStackView() {
+        categoryStackView.addArrangedSubviews(
+            categoryTagView,
+            dividerImageView,
+            categoryDetailLabel
+        )
+        
+        writerInfoStackView.addArrangedSubviews(
+            writerProfileImageView,
+            writerNameLabel
+        )
+        
+        contentStackView.addArrangedSubviews(
+            categoryStackView,
+            writerInfoStackView,
+            titleLabel,
+            contentLabel,
+            coverImageView
+        )
+    }
+}
+
+// MARK: - Methods
+
+extension AnnouncementCardCVC {
+    func configureCell(model: AnnouncementInfo) {
+        self.categoryTagView.setData(with: model.categoryName, isHotTag: false)
+        self.categoryDetailLabel.text = model.categoryDetailName
+        self.writerProfileImageView.setImage(with: model.profileImage)
+        self.writerNameLabel.text = model.name
+        self.titleLabel.text = model.title
+        self.contentLabel.text = model.content
+        
+        let hasCoverImage = (model.images != nil)
+        updateContentLayout(hasCoverImage: hasCoverImage)
+        if hasCoverImage, let cover = model.images {
```

Comment:
model.images가 nil이면 if문도 실행되지 않으니 hasCoverImage는 불필요해보여요!
```suggestion
        if let cover = model.images {
```